tests/pytests/unit/modules/test_openscap.py

```diff
@@ -0,0 +1,421 @@
+import subprocess
+
+import pytest
+
+import salt.modules.openscap as openscap
+from tests.support.mock import MagicMock, Mock, patch
+
+random_temp_dir = "/tmp/unique-name"
+policy_file = "/usr/share/openscap/policy-file-xccdf.xml"
+
+
+class OpenscapTestMock:
+    def __init__(self):
+        import salt.modules.openscap
+
+        salt.modules.openscap.__salt__ = MagicMock()
+        self.patchers = [
+            patch("salt.modules.openscap.__salt__", MagicMock()),
+            patch("salt.modules.openscap.shutil.rmtree", Mock()),
+            patch(
+                "salt.modules.openscap.tempfile.mkdtemp",
+                Mock(return_value=random_temp_dir),
+            ),
+            patch("salt.modules.openscap.os.path.exists", Mock(return_value=True)),
+        ]
+        for patcher in self.patchers:
+            self.apply_patch(patcher)
+
+    def apply_patch(self, patcher):
+        patcher.start()
```

Comment:
@s0undt3ch are you okay with using `patcher.start()` here since it's also handling the close?

Reply:
@s0undt3ch @Ch3LL friendly reminder here, so we don't leave this PR pending for long time again.

Thanks in advance!